migration/1_init.sql

```diff
@@ -1,5 +1,6 @@
--- CREATE TABLE "user" ----------------------------------------
-CREATE TABLE IF NOT EXISTS "user" (
```

Comment:
Filename can be another. It's not must have thing to call the first migration as init.
If it's a little migration and we can find a better name - let's do it.

In our case, we can call it like - "001_create_user_table"

Reply:
Fixed.